### ISSUE TITLE
No ActorIdentifier rows for GCOM_ID

### ISSUE THREAD

**Issue by evanp** [Open-Earth-Foundation/OpenClimate-harmonize#13]
Could you please add a mapping of GCOM_ID to actor_id in the ActorIdentifier table? It seems a waste to throw away those mappings.

**Comment by lgloege** [comment]
Of course! RIght now I stuck the GCOM_ID in the `emissions_id`, with the format `<datasource>:<GCOM_ID>:<year>`

**Comment by evanp** [comment]
OK, so what I want is an ActorIdentifier.csv with entries like this:

| actor_id | identifier | namespace |
|--------- |----------|------------|
| IT-LAI      | IT1516   | GCOMID    |
| BE-WJG | BE0211.  | GCOMID    |
